### DISCUSSION TITLE
How to plot partition asset myself ?

### DISCUSSION THREAD

**Discussion by dridk** [dagster-io/dagster#22815] [Q&A]
Hi,

I would like to extract data from dagster instance to plot the following chart myself. ( as a png file somewhere )
![img](https://user-images.githubusercontent.com/49863538/222094787-1ea0ddc5-36f9-4553-8080-8a4ce2a96d45.png)

For instance, the following pseudo-code : 
```python

# pseudo-code 

# >>> ??
for asset in dagster.instance.get_all_asset():
     for partition in asset:
           x = partition.get_partition_name()
           y = partition.get_partition_metadata("row_count")
           data.append((x,y))
# <<< ??
df = pd.DataFrame(data)
sns.plot("x", "y", df)

```

Do you have some hint or code snippet to share ?  Or you may point me to the source code where the data plot come from.

**Comment by garethbrickman** [comment]
I can suggest using your browser's Developer Tools to inspect the chart element and to also see the network requests being run. It's likely pulling data from one or more GraphQL queries
